### PR TITLE
Add the version to footer

### LINKED ISSUE #988: [FEATURE] Add version to the footer
**Is your feature request related to a problem? Please describe.**
Add the current version to the footer

**Describe the solution you'd like**
Read it from the git tag

**Describe alternatives you've considered**
Hard code it to HTML

in config.py
    repo = Repo(os.path.dirname(os.path.realpath(__file__)))
    gitinfo = "{}".format(repo.tags[-1])
    session['gitinfo'] = gitinfo

in jinjia 2 template:
{{session['username']}}

### PR DESCRIPTION
**Pull request by clav1d**
## Description
_Please provide a summary of the pull request and the issue it fixes. Please add necessary details, context, dependencies, explanation of when review is needed (see next section), etc._

**Fixes #988**

[comment]: # (This project only accepts pull requests related to open issues.)
[comment]: # (If suggesting a new feature or change, please discuss it in an issue first.)
[comment]: # (If fixing a bug, there should be an issue describing it with steps to reproduce.)

## Review Time Estimate
_Please give your idea of how soon this pull request needs to be reviewed by selecting one of the options below. This can be based on the criticality of the issue at hand and/or other relevant factors._

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Immediately
- [ ] Within a week
- [x] When possible

## Type of changes
_Please select a relevant option:_

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Bug fix (non-breaking change which fixes an issue).
- [x] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] Other (any another change that does not fall in one of the above categories.)

## Checklist:
_Please select all applicable options:_

[comment]: # (To select your options, please put an 'x' in the all boxes that apply.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] I have signed the Rokwire Contributor License Agreement ([CLA](https://rokwire.org/rokwire_cla)). (_Any contributor who is not an employee of the University of Illinois whose official duties include contributing to the Rokwire software, or who is not paid by the Rokwire project, needs to sign the CLA before their contribution can be accepted._)
- [x] I have updated the [CHANGELOG](../CHANGELOG.md).
- [ ] I have read the [Contributor Guidelines](../CONTRIBUTING.md).
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] My change requires updating the documentation.
- [ ] I have made necessary changes to the documentation.
- [ ] I have added tests related to my changes.
- [ ] My changes generate no new warnings.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules.

---

[comment]: # (Template credit: This pull request template is based on Embedded Artistry {https://github.com/embeddedartistry/templates/blob/master/.github/PULL_REQUEST_TEMPLATE.md}, Clowder {https://github.com/clowder-framework/clowder/blob/develop/.github/PULL_REQUEST_TEMPLATE.md}, and TalAter {https://github.com/TalAter/open-source-templates} templates.)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Update the style of buttons on the event detail page. [#984](https://github.com/rokwire/events-manager/issues/984)
 - Add elements to the footer. [#969](https://github.com/rokwire/events-manager/issues/969)
 - Update the GUI of the campus calendar listing page. [#972](https://github.com/rokwire/events-manager/issues/972)
+- Add the version to footer. [#988](https://github.com/rokwire/events-manager/issues/988)
```
Comment (sandeep-ps) [inline CHANGELOG.md:25]: Please move this entry to the Unreleased section.

**Review by sandeep-ps — APPROVED** [review]
Approving PR with changes for CHANGELOG.

**Comment by bingzhang** [comment]
@clav1d can you quick fix the changelog? then we can merge. thanks.

**Comment by minump** [comment]
Change text color to white and have the same spacing between the 3 texts.

Screen Shot 2022-12-16 at 3 46 53 PM

**Comment by bingzhang** [comment]
> Change text color to white and have the same spacing between the 3 texts.
> 
> Screen Shot 2022-12-16 at 3 46 53 PM

@clav1d can you fix the comment? thanks.

**Comment by clav1d** [comment]
> Change text color to white and have the same spacing between the 3 texts.
> 
> Screen Shot 2022-12-16 at 3 46 53 PM

I'm not able to reproduce this bug. Can you tell me your browser's version and the steps to reproduce this bug?

**Review by bingzhang — APPROVED** [review]

**Pull request merged**